### PR TITLE
Vendor in latest containers image

### PR DESCRIPTION
**Pull request by rhatdan**
We also want to add the containers/image/docs directory to the
vendor so that the skopeo rpm can pull the man pages for registries.conf
and policy.json directly from the repo.

Added new command to Makefile vendor-image, that will vendor in the latest
container/image, and maintain its docs directory.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@runcom @mtrmac PTAL

File: Makefile [hunk 1]
```diff
@@ -114,6 +114,9 @@ install-completions:
 shell: build-container
 	$(DOCKER_RUN_DOCKER) bash
 
+vendor-image:
+	vndr -whitelist '^github.com/containers/image/docs/.*' github.com/containers/image
```
Comment (mtrmac) [inline Makefile]: Overall, `README.md` needs updating as well.

I’m worried having a special command for vendoring c/image (as opposed to anything else) will get overlooked, though.

Maybe
```make
vendor: vendor.conf
	vndr -whitelist '^github.com/containers/image/docs/.*'
```
and then document `make vendor`?  OTOH PRs frequently update only the c/image subpackage (people understandably don’t want to be on the hook for changes in other packages they don’t care for).

I don’t know what’s the usual practice.

Comment (rhatdan) [reply Makefile]: How about we add a test that will fail if the docs are not there and report an error explaining that you should execute make vendor if you are changing containers/image.

Comment (rhatdan) [reply Makefile]: I updated the README.md, also added test to check for docs.

**Comment by rhatdan** [comment]
@runcom @mtrmac I added changes to the README and a test which will fail if some one vendors in containers/image without using the special Makefile command.
PTAL

**Comment by mtrmac** [comment]
ACK.

I’m not _thrilled_ about treating c/image specially here, but it does work well enough.

Please drop `hack/vendor.sh` (it would be another place to update, and we don’t use it nor AFAICT need it for anything) and rebase.

**Comment by rhatdan** [comment]
@mtrmac Fixed the way you wanted.
@runcom PTAL

**Comment by rhatdan** [comment]
Shoot I just blew my patch away.

**Pull request closed without merging**